src/WebJobs.Extensions.DurableTask/Options/DurableDistributedTracingVersion.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Options
+{
+    /// <summary>
+    /// Represents configuration options for Distributed Tracing in Durable Functions.
+    /// </summary>
+    public enum DurableDistributedTracingVersion
+    {
+        /// <summary>
+        /// Original implementation of Distributed Tracing in Durable Functions
+        /// that can be configured to use HttpCorrelationProtocol or W3CTraceContext.
+        /// </summary>
+        V1 = 0,
```

Comment:
We should probably add a `None` value, just to set us up to remove the explicit enabled property on the options (or at least tweak it to interact with this enum, and in a future PR we can do this).

Make sure to then also update the `TelemetryActivator` to no-op when `None` is set.

``` CSharp
None = 0,
V1 = 1,
V2 = 2,
```